src/Insect/Parser.purs

```diff
@@ -191,6 +191,11 @@ normalUnitDict ∷ Dictionary DerivedUnit
 normalUnitDict = Dictionary
   [ Q.radian ==> ["radians", "radian", "rad"]
   , Q.degree ==> ["degrees", "degree", "deg", "°"]
+  , Q.pct ==> ["pct", "percent"]
+  , Q.ppm ==> ["ppm"]
+  , Q.ppb ==> ["ppb"]
+  , Q.ppt ==> ["ppt"]
+  , Q.ppq ==> ["ppq"]
```

Comment:
We have to be a bit careful here.

If we add these units in `normalUnitDict`, combinations with SI prefixes will be allowed. Something like `kppm` (kilo-parts-per-million). I don't think we want that :smile: 

I think everything should work fine if we add it to `imperialUnitDict` below (the name should really be changed).

Reply:
Makes sense. Seems to work fine. Updated.